src/refactor_bot/cli/main.py

```
@@ -31,7 +34,8 @@
 _SAFE_CONFIG_KEYS = frozenset({
     "directive", "repo_path", "max_retries", "model",
     "vector_store_dir", "timeout", "verbose", "dry_run", "output_json",
-    "skills", "allow_no_runner_pass",
+    "skills", "allow_no_runner_pass", "llm_provider", "llm_fallback_provider",
+    "allow_llm_fallback", "allow_no_runner_pass", "interactive_fallback",
```

Ruff (0.15.1)

[warning] Sets should not contain duplicate item `"allow_no_runner_pass"`

Remove duplicate item

(B033)